mdstcpip/mdsip-client-http

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python -u
+#!/usr/bin/env python -u
```

Comment:
So why did you change the first one from `/usr/bin/env python` to `/usr/bin/python2` but the rest of them you changed to have `/usr/bin/env`?